release/ray_release/scripts/run_release_test.py

```diff
@@ -73,7 +74,10 @@
 @click.option(
     "--env",
     default=None,
-    type=click.Choice(["prod", "staging"]),
+    # Get the names without suffixes of all files in "../environments"
+    type=click.Choice(
+        [x.stem for x in (Path(__file__).parent.parent / "environments").glob("*.env")]
+    ),
```

Comment:
why do we need this?

Reply:
This allows the script user to change the environment for the test - previously this was hardcoded, now it matches all available environment files